database/data.js

```diff
@@ -0,0 +1,87 @@
+var faker = require('faker');
+var csv = require('csv-parser');
+var fs = require('fs');
+
+//set seed
+faker.seed(123);
+
+var generateRating = function(max){
+  // return random Number between 0 and max
+  var randomNum = parseInt(Math.random() * max)
```

Comment:
I believe this function returns a random number btwn 0 and max-1, not 0 and max.

Reply:
I wasn't using this function so I just removed it